### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by UvinduRajapakshe** [AbirHasan2005/Pyro-FileStreamBot#3] [Q&A] [answered]
```
2021-04-22T08:17:13.580780+00:00 heroku[web.1]: Idling
2021-04-22T08:17:13.583097+00:00 heroku[web.1]: State changed from up to down
2021-04-22T08:17:14.637987+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2021-04-22T08:17:16.548324+00:00 heroku[web.1]: Process exited with status 0
```

Bot Starting successfully After the sometimes bot down with this error

**Comment by AbirHasan2005** [comment]
That's Not a Error, that's because of Heroku `web` Server Sleeps after 30 minutes. You can Ping if to Keep it always Active. I am also doing same thing.

If need more help you can ask in [Support Group](https://t.me/linux_repo).

**Reply by UvinduRajapakshe** [reply]
Okay 😊 But, Can you tell me Wow to ping it to keep alive ?

**Reply by AbirHasan2005** [reply]
https://kaffeine.herokuapp.com/

Add Heroku App Name and Start Pinging.

**Reply by UvinduRajapakshe** [reply]
Got it Thank You ❤

**Comment by UvinduRajapakshe** [comment, marked as answer]
Why is this 🤕

![image](https://user-images.githubusercontent.com/79355885/116089940-fa684080-a6c0-11eb-8127-b94343edcda3.png)

**Reply by AbirHasan2005** [reply]
Your MongoDB URI Not Valid!

Check a Tutorial [Here](https://t.me/linux_repo/282294).
